Userland/Libraries/LibJS/Runtime/RegExpPrototype.cpp

```diff
@@ -346,7 +346,7 @@ JS_DEFINE_NATIVE_FUNCTION(RegExpPrototype::symbol_replace)
         size_t result_length = length_of_array_like(global_object, result);
         size_t n_captures = result_length == 0 ? 0 : result_length - 1;
 
-        auto matched = result.get(0).value_or(js_undefined());
+        auto matched = result.get(0).value_or(js_undefined()).to_string(global_object);
```

Comment:
This cannot be in the same expression but needs to be split into multiple lines unfortunately - get can throw, and `to_string()` can call a JS function which will go terribly wrong (:boom:) if we already have an exception stored.

Reply:
Maybe `get()` should return an `ExceptionOr`?
Would then force people to check for exceptions

Reply:
fixed! thanks for the test as well :)